packages/libs/eda/src/lib/map/analysis/hooks/standaloneVizPlugins.ts

```diff
@@ -40,6 +40,10 @@ export function useStandaloneVizPlugins({
       return visualization.withOptions({
         hideFacetInputs: true, // will also enable table-only mode for mosaic
         hideShowMissingnessToggle: true,
+        // TODO: need to distinguish lineplot from lineplot with marginal histogram?
+        // perhaps need to make another function only for lineplot with marginal histogram
+        // or define xxx.withOptions({}) directly at corresponding visualizationPlugins below
```

Comment:
Yes I think we need to add `showMarginalHistogram` for just the appropriate lineplot(s).